components/Frame/index.js

```diff
@@ -23,9 +23,7 @@ import {
 import { withMembership } from '../Auth/checkRoles'
 import withMe from '../../lib/apollo/withMe'
 import withT from '../../lib/withT'
-import withInNativeApp, {
-  inNativeAppBrowserLegacy
-} from '../../lib/withInNativeApp'
+import withInNativeApp, { useInNativeApp } from '../../lib/withInNativeApp'
```

Comment:
If you use the hook, you might as well remove `withInNativeApp` and just use the hook.